test/distributed/fsdp/test_distributed_checkpoint.py

```diff
@@ -7,8 +7,8 @@
 from torch.distributed.checkpoint import (
     FileSystemReader,
     FileSystemWriter,
-    load_state_dict,
-    save_state_dict,
+    load,
+    save,
```

Comment:
is this a name change or is there functionality change?
cc @fegin

Reply:
Mostly a name change and API signature change.